includes/admin/home/notices/class-sensei-home-notices-provider.php

```diff
@@ -81,10 +81,13 @@ public function get( $max_age = HOUR_IN_SECONDS ): array {
 	 */
 	public function get_badge_count(): int {
 		add_filter( 'sensei_home_remote_data_retry_error', '__return_false' );
-		$notices = $this->get( DAY_IN_SECONDS );
+		$notices                 = $this->get( DAY_IN_SECONDS );
+		$notices_with_parent_ids = array_column( $notices, 'parent_id' );
+		$notices_with_parent_ids = array_filter( $notices_with_parent_ids );
```

Comment:
Maybe it would be clearer to just do `$notices_with_parent_ids = array_filter( array_column( $notices, 'parent_id' ) );` rather than reassigning the variable? 🤷‍♂️

Reply:
Sure, done in [1701b7e](https://github.com/Automattic/sensei/pull/6274/commits/1701b7e2f6f6a1a98c54ed748e0e514c0e315bf1). Thanks!